tools/cluster_test_rule.bzl

```diff
@@ -88,7 +91,10 @@ def _rule_implementation(ctx):
     cmd += """
            echo Tests concluded with exit value $RESULT
            echo Stopping servers.
-           kill $(jps | awk '/TypeDBNode/ {print $1}' | paste -sd " " -)
+           procs=$(jps | awk '/TypeDBNode/ {print $1}' | paste -sd " " -)
+           if [ -n "$procs" ]; then
+             kill $procs
+           fi
```

Comment:
note to future self: we needed to add this because the cluster failover test can kill the servers started by this script and restart them from within the test. This means by the time we get to `kill` all the original TypeDB processes will already be dead and the `kill` command will fail because there are no processes left